schemas.sql

```diff
@@ -0,0 +1,112 @@
+
+-- keep track of all users (id, tag, name)
+
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    user_id VARCHAR NOT NULL,
+    user_tag VARCHAR NOT NULL,
+    user_name VARCHAR NOT NULL,
+    deleted_at TIMESTAMP
+);
+
+-- From rememberMessage.js
+-- 'content' is a VARCHAR and not TEXT because 
+-- discord has a limit of 2000 chars per message (4000 for nitro users)
+-- and VARCHAR has a limit of 10485760
+CREATE TABLE messages (
+    id SERIAL PRIMARY KEY,
+    user_id VARCHAR NOT NULL,
+    content VARCHAR NOT NULL,
+    message_id VARCHAR NOT NULL,
+    message_ts VARCHAR NOT NULL,
+    deleted_at TIMESTAMP
+);
+
+-- keep in mind, array size is just verbose and effectively does nothing
+CREATE TABLE checkin_schedules (
+    id SERIAL PRIMARY KEY,
+    user_id VARCHAR NOT NULL,
+    utc_days VARCHAR[],
+    utc_time INT[2],
+    local_days VARCHAR[],
+    local_time INT[2],
+    deleted_at TIMESTAMP
+);
```

Comment:
For the two schedules tables, (if the messages table is storing discord messages) I might add a reference to `messages` just to associate a schedule with the message it was sourced from.